Gemfile.lock

```diff
@@ -142,36 +141,36 @@ GEM
     concurrent-ruby (1.1.5)
     connection_pool (2.2.2)
     content_for_in_controllers (0.0.2)
-    coveralls (0.8.22)
-      json (>= 1.8, < 3)
-      simplecov (~> 0.16.1)
-      term-ansicolor (~> 1.3)
-      thor (~> 0.19.4)
-      tins (~> 1.6)
+    coveralls (0.7.2)
```

Comment:
this looks more link an downgrade than upgrade :joy:

Reply:
that is weird ... thanks for catching this!

Reply:
I've tried to correct this, but there must be some combination of dependencies that doesn't work.
```
Bundler attempted to update coveralls but its version stayed the same
```

[`bundle-stats`](https://olivierlacan.com/posts/why-wont-bundle-update/) may be of help here.
[select2-rails](https://rubygems.org/gems/select2-rails) has `thor ~> 0.14` while the newer coveralls has `thor ~> 0.19.4` (from [thor 0.7.2](https://rubygems.org/gems/coveralls/versions/0.7.2) there is a thor version requirement). I guess we'll need a new select2 dependency :(

Reply:
IMHO there is no need to spend any time on this. just merge it